### ISSUE TITLE
Missing licence

### ISSUE THREAD

**Issue by fishnux** [jar-o/limlog#2]
Hi,

Could you please specify a licence for this project?

Nice work, thank you!

**Comment by MarcSchunk** [comment]
yes, pls add a license, this would allow more pls to use your package
